core/block/import/syncer/file.go

```diff
@@ -50,6 +49,5 @@ func (fs *FileSyncer) Sync(ctx *session.Context, id string, b simple.Block) erro
 	if err != nil {
 		return fmt.Errorf("failed syncing file: %s", err)
 	}
-	os.Remove(b.Model().GetFile().Name)
```

Comment:
Remove it, because it's not safe to remove files on user local machine. Originally we delete file after its loading, because in Markdown import we copy media files to temporary directory and then load them. So we try to remove file from temporary directory here after loading. But its common code for all import now, and there is a probability, that file Name contain path to not temporary file on local machine. And this code can delete some important files in theory